### ISSUE TITLE
Add Safezone support

### ISSUE THREAD

**Issue by BlackFisch** [kayler-renslow/arma-dialog-creator#29]
### Version and Build
1.0.5

### Summary
native support for creating dialogs

### Detail
(please correct me, if I just id not see the feature)
It would be nice to have a feature for creating relative dialogs via Safezone instead having to add it manually to the absolute positions of the controls. Would make life easier a little bit ;)

**Comment by BlackFisch** [comment]
As you might see - I'm pretty stupid when it comes to finding functions and understanding the program xD Did not realize, that Safezone gets added automatically when rescaling the control xD

**Comment by kayler-renslow** [comment]
You can currently double click controls in the editor to open up the Control Properties Editor

**Comment by BlackFisch** [comment]
Yeah, just found that out ^^ Honestly, i miss the <DEL>-Hotkey the most ^^
